clusters/playground/overlay/third-party/snyk-monitor/helmRelease.yaml

```diff
@@ -6,4 +6,11 @@ metadata:
 spec:
   chart:
     spec:
-      version: 1.77.2 # https://artifacthub.io/packages/helm/snyk/snyk-monitor
+      version: 1.80.0 # https://artifacthub.io/packages/helm/snyk/snyk-monitor
+  values:
+    requests:
+      cpu: "1"
+      memory: "2Gi"
+    limits:
+      cpu: "2"
+      memory: "4Gi"
```

Comment:
Suggest to keep limits and requests order similar for different configurations - harder to make mistakes. E.g. first `limits`, then `requests`